src/components/duelling-picklist/duelling-picklist.stories.mdx

```diff
@@ -52,32 +56,119 @@ It invokes passed `onChange` method with passed `item` prop as an argument when
 ## Examples
 
 ### Default
+
 Example of composed Duelling Picklist components with order preservation and search implemented.
 
 <Preview>
   <Story id="test-duellingpicklist--default" name="Default" />
 </Preview>
 
-### In dialog
+### Grouped
+
+Picklist items can be organised into selectable groups. Whole groups can be moved from one list to another at once, or group items can be moved individually.
+
+<Preview>
+  <Story id="test-duellingpicklist--grouped" name="Grouped" />
+</Preview>
+
+### In Dialog
+
 Same as above but as a children of `Dialog` component.
 
 <Preview>
   <Story id="test-duellingpicklist--in-dialog" name="InDialog" />
 </Preview>
 
-## Props
+## PicklistItem Examples
+
+### Add item
+
+<Preview>
+  <Story name="add item">
+    <PicklistItem type="add" item={1} onChange={() => null}>
+      <div style={{ display: "flex", width: "100%" }}>
+        <div style={{ width: "50%" }}>
+          <p style={{ fontWeight: 700, margin: 0, marginLeft: 24 }}>
+            Title for Item
+          </p>
+        </div>
+      </div>
+    </PicklistItem>
+  </Story>
+</Preview>
+
+### Remove item
+
+<Preview>
+  <Story name="remove item">
+    <PicklistItem type="remove" item={1} onChange={() => null}>
+      <div style={{ display: "flex", width: "100%" }}>
+        <div style={{ width: "50%" }}>
+          <p style={{ fontWeight: 700, margin: 0, marginLeft: 24 }}>
+            Title for Item
+          </p>
+        </div>
+      </div>
+    </PicklistItem>
+  </Story>
+</Preview>
+
+### Locked
+
+<Preview>
+  <Story name="locked">
+    <PicklistItem type="add" item={1} onChange={() => null} locked>
+      <div style={{ display: "flex", width: "100%" }}>
+        <div style={{ width: "50%" }}>
+          <p style={{ fontWeight: 700, margin: 0, marginLeft: 24 }}>
+            Title for Item
+          </p>
+        </div>
+      </div>
+    </PicklistItem>
+  </Story>
+</Preview>
+
+### Custom tooltip message
+
+<Preview>
+  <Story name="custom tooltip message">
+    <PicklistItem
+      type="add"
+      item={1}
+      onChange={() => null}
+      locked
+      lockedTooltipMessage="This is a custom locked tooltip message"
+    >
+      <div style={{ display: "flex", width: "100%" }}>
+        <div style={{ width: "50%" }}>
+          <p style={{ fontWeight: 700, margin: 0, marginLeft: 24 }}>
+            Title for Item
+          </p>
+        </div>
+      </div>
+    </PicklistItem>
+  </Story>
+</Preview>
+
+## Props:
 
 ### Picklist
+
 <Props of={Picklist} />
 
 ### PicklistItem
+
 <Props of={PicklistItem} />
 
 ### DuellingPicklist
+
 <Props of={DuellingPicklist} />
 
 ### PicklistDivider
+
 <Props of={PicklistDivider} />
```

Comment:
Props table isn't being created for `PicklistDivider`.

<img width="1266" alt="Screenshot 2021-03-23 at 11 34 51" src="https://user-images.githubusercontent.com/56251247/112140476-f5990400-8bcb-11eb-9f6b-8a2c34fae744.png">

Reply:
It doesn't actually have any props, I'll just remove it from the props section